kiwi_lint/__init__.py

```diff
@@ -14,6 +16,17 @@
 from .empty_module import EmptyModuleChecker
 from .nested_definition import NestedDefinitionChecker
 from .missing_permissions import MissingPermissionsChecker
+from .function_based_views import FunctionBasedViewChecker
+
+
+def has_django_installed():
+    """Django must be installed and DJANGO_SETTINGS_MODULE must be present in shell environment."""
+    try:
+        __import__('django')
+    except (ImportError, ModuleNotFoundError):
+        return False
+
+    return 'DJANGO_SETTINGS_MODULE' in os.environ
```

Comment:
Since this plugin is only valid in the current project and we are fairly certain Django is always installed we don't really need this function.